### PR TITLE
add xapi.us

### PR DESCRIPTION
**Pull request by djekl**

### CODE REVIEW

File: img/x/xapi.us.svg [hunk 1]
```diff
@@ -0,0 +1,7 @@
...
+      fill="currentColor"/>
+</svg>
```
Comment (Carlgo11) [inline img/x/xapi.us.svg:7]:
```suggestion
<svg viewBox="0 0 20 20" xmlns="http://www.w3.org/2000/svg"><path fill="#fe5000" d="m10 6.84 1.05-3.16a1 1 0 0 1 1.9.64L11.05 10l1.9 5.68a1 1 0 0 1-1.9.64L10 13.16l-1.05 3.16a1 1 0 0 1-1.9-.64L8.95 10l-1.9-5.68a1 1 0 0 1 1.9-.64L10 6.84ZM5.02 6l.12.01h.03l.02.01h.02l.03.01c.17.04.33.13.47.26a1 1 0 0 1 0 1.42L3.4 10l2.3 2.3a1 1 0 0 1-.22 1.59l-.02.01-.03.01-.02.01a.96.96 0 0 1-.17.06L5.2 14l-.03.01H5.1l-.03.01H4.9a.92.92 0 0 1-.13-.03h-.02l-.03-.01-.02-.01h-.02l-.03-.02h-.02l-.02-.02H4.6a1 1 0 0 1-.3-.21l-3-3a1 1 0 0 1-.1-1.28V9.4l.02-.02.02-.02v-.02c.02 0 .02-.02.03-.03l.03-.03 3-3A1 1 0 0 1 4.87 6h.16ZM15 6h.12v.01c.22.03.42.12.59.28l3 3a.93.93 0 0 1 .17.24c.07.12.1.26.12.4v.13l-.01.08a1 1 0 0 1-.28.57l-3 3c-.02 0-.03.02-.04.03a1.2 1.2 0 0 1-.08.07l-.02.01a.7.7 0 0 1-.24.13h-.02l-.05.02-.04.01a1 1 0 0 1-.13.03h-.03a.6.6 0 0 1-.06 0h-.16L14.8 14h-.02l-.05-.02a1 1 0 0 1-.45-1.69L16.6 10l-2.3-2.3a1 1 0 0 1 .22-1.57l.02-.01.03-.01a.93.93 0 0 1 .15-.07h.05l.03-.01.04-.01h.03l.11-.02H15Z"/></svg>
```

File: entries/x/xapi.us.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+  "XAPI": {
+    "domain": "xapi.us",
+    "tfa": [
+      "totp"
+    ],
+    "keywords": [
+      "developer",
+      "gaming"
```
Comment (Carlgo11) [inline entries/x/xapi.us.json]: Whilst the service does seem to integrate with a gaming service I wouldn't classify XAPI as a gaming service.
```suggestion
      "developer"
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

**Comment by Carlgo11** [comment]
Just need to confirm with you that you've read the CONTRIBUTING.md sections relevant to this PR. Most importantly the guidelines.

**Comment by djekl** [comment]
> Just need to confirm with you that you've read the CONTRIBUTING.md sections relevant to this PR. Most importantly the guidelines.

I had read them yes, but I missed the guideline about the top 200k. Unfortunately, upon review, my site does not meet this requirement guideline.

The reason I wish to have my site listed in here, is so that 1Password picks up the 2FA availability.

If, because of the Similarweb ranking, I am not able to be listed. Then I understand.

**Comment by Carlgo11** [comment]
Like you say, xapi.us does not meet the 4th site criteria and thus can't be listed.
If your site meets the criteria in the future, you can re-open this PR and we'll have a look at it then.

Apologies for the inconvenience.

**Pull request closed without merging**